infra/resources/_modules/function_apps/locals.tf

```diff
@@ -1,50 +1,69 @@
+    PidIssuerHealthCheckEnabled   = true
 
-      RevocationQueueName = var.revocation_queue_name
+    RevocationQueueName = var.revocation_queue_name
 
-      WEBSITE_SWAP_WARMUP_PING_PATH     = "/api/v1/wallet/health"
-      WEBSITE_SWAP_WARMUP_PING_STATUSES = "200"
-      },
-      local.function_apps.common_app_settings,
+    WEBSITE_SWAP_WARMUP_PING_PATH     = "/api/v1/wallet/health"
+    WEBSITE_SWAP_WARMUP_PING_STATUSES = "200"
+    },
+    local.function_apps.common_app_settings,
+    {
+      for s in var.user_func.app_settings :
+      s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.key_vault_wallet_name};SecretName=${s.key_vault_secret_name})" : s.value
+    }
+  )
+
+  function_app_user_staging_disabled = [
```

Comment:
I think you should put the entire variable name like `AzureWebJobs.addWalletInstanceToValidationQueue.Disabled`

Reply:
below I put `format("AzureWebJobs.%s.Disabled", to_disable) => false`